src/Tizen.Multimedia.Remoting/MediaController/MediaControlCommand.cs

```diff
@@ -0,0 +1,271 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Tizen.Applications;
+using System;
+using Native = Interop.MediaControllerClient;
+using NativeHandle = Interop.MediaControllerClientHandle;
+
+namespace Tizen.Multimedia.Remoting
+{
+    /// <summary>
+    /// Provides a means to send command to media control server.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public abstract class Command
+    {
+        internal NativeHandle _receiverHandle;
+        internal string _serverId;
+
+        internal Command(MediaControlCommand type)
+        {
+            Type = type;
+        }
+
+        internal MediaControlCommand Type { get; }
+
+        internal abstract string Send();
+
+        internal void SetReceiver(NativeHandle receiverHandle, string serverId)
+        {
+            _receiverHandle = receiverHandle;
+            _serverId = serverId;
+        }
+    }
+
+    /// <summary>
+    /// Provides a means to send playback command to media control server.
+    /// </summary>
+    public sealed class PlaybackCommand : Command
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PlaybackCommand"/> class.
+        /// </summary>
+        /// <param name="command">A <see cref="MediaControlPlaybackCommand"/>.</param>
+        /// <since_tizen> 5 </since_tizen>
+        public PlaybackCommand(MediaControlPlaybackCommand command)
+            : base(MediaControlCommand.Playback)
+        {
+            Command = command;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PlaybackCommand"/> class.
+        /// </summary>
+        /// <param name="command">A <see cref="MediaControlPlaybackCommand"/>.</param>
+        /// <param name="playlistName">The playlist name of the server.</param>
+        /// <param name="index">The index of the media in the playlist.</param>
+        /// <param name="position">The playback position in milliseconds.</param>
+        /// <exception cref="ArgumentException"><paramref name="index"/> cannot be converted to number.</exception>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="playlistName"/> or <paramref name="index"/> is not vailed.
+        /// </exception>
+        /// <since_tizen> 5 </since_tizen>
+        public PlaybackCommand(MediaControlPlaybackCommand command, string playlistName, string index, ulong position)
```

Comment:
Why index type is string?

Reply:
It can be any string as index not just integer number.

Reply:
>It can be any string as index not just integer number.
It is not true, see below code
```
            if (!Int32.TryParse(PlaylistIndex, out int result))
            {
                throw new ArgumentException("Playlist index is not vaild to convert to number.");
            }
```

Reply:
It's deleted in my local patch. I'll update it in next commit.